menu.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Menu</title>
+</head>
+<body>
+  <section>
+    <div class="container">
+      <h1>Our Menu</h1>
+      <p>We consider all the drivers of change gives you the components you need to change to create a truly happens.</p>
+    </div>
+  </section>
+  <section>
+    <div class="container">
+      <div class="filer-container">
+        <button>All</button>
+        <button>Breakfast</button>
+        <button>Main Dishes</button>
+        <button>Drinks</button>
+        <button>Desserts</button>
+      </div>
+      <div class="cards-container">
+        <div class="card">
+          <img src="images/menu1.png" alt="Fried Eggs">
+          <span>&dollar;9.99</span>
+          <h2>Fried Eggs</h2>
+          <p>Made with eggs, lettuce, salt, oil and other ingredients.</p>
+        </div>
+        <div class="card">
+          <img src="images/menu2.png" alt="Hawaiian Pizza">
+          <span>&dollar;15.99</span>
+          <h2>Hawaiian Pizza</h2>
+          <p>Made with eggs, lettuce, salt, oil and other ingredients.</p>
+        </div>
+        <div class="card">
+          <img src="images/menu3.png" alt="Martinez Cocktail">
+          <span>&dollar; 7.25</span>
+          <h2>Martinez Cocktail</h2>
+          <p>Made with eggs, lettuce, salt, oil and other ingredients.</p>
+        </div>
+        <div class="card">
+          <img src="images/menu4.png" alt="Butterscotch Cake">
+          <span>&dollar; 20.99</span>
+          <h2>Butterscotch Cake</h2>
+          <p>Made with eggs, lettuce, salt, oil and other ingredients.</p>
+        </div>
+        <div class="card">
+          <img src="images/menu5.png" alt="Mint Lemonade">
+          <span>&dollar; 5.89</span>
+          <h2>Mint Lemonade</h2>
+          <p>Made with eggs, lettuce, salt, oil and other ingredients.</p>
+        </div>
+        <div class="card">
+          <img src="images/menu6.png" alt="Chocolate Icecream">
+          <span>&dollar; 18.05</span>
+          <h2>Chocolate Icecream</h2>
+          <p>Made with eggs, lettuce, salt, oil and other ingredients.</p>
+        </div>
+        <div class="card">
+          <img src="images/menu7.png" alt="Cheese Burger">
+          <span>&dollar; 12.55</span>
+          <h2>Cheese Burger</h2>
+          <p>Made with eggs, lettuce, salt, oil and other ingredients.</p>
+        </div>
+        <div class="card">
+          <img src="images/menu8.png" alt="Classic Waffles">
+          <span>&dollar; 9.99</span>
+          <h2>Classic Waffles</h2>
+          <p>Made with eggs, lettuce, salt, oil and other ingredients.</p>
+        </div>
+      </div>
+    </div>
+  </section>
+  <section>
+    <div class="container">
```

Comment:
we do not need two containers in the page only one with this structure 
<section>
  <container>
   <section></section>
<section></section>
</container>
</section?